presto-main/pom.xml

```diff
@@ -335,6 +335,12 @@
             <artifactId>jaxrs-testing</artifactId>
```

Comment:
can you please add configuration variable so encryption could be enabled or disabled depending on user needs